phoenix5-spark/README.md

```diff
@@ -45,12 +57,14 @@ val spark = SparkSession
 val df = spark.sqlContext
   .read
   .format("phoenix")
-  .options(Map("table" -> "TABLE1"))
+  .option("table", "TABLE1")
```

Comment:
Nit:
Is the is the preferred way to add options in spark ?
Are you supposed to call the method multiple times if you want to add multiple options ?